plugins/thunderbolt/fu-thunderbolt-device.c

```diff
@@ -331,14 +338,76 @@ fu_thunderbolt_device_setup (FuDevice *device, GError **error)
 		fu_device_add_flag (device, FWUPD_DEVICE_FLAG_USABLE_DURING_UPDATE);
 		/* forces the device to write to authenticate on disconnect attribute */
 		fu_device_remove_flag (device, FWUPD_DEVICE_FLAG_SKIPS_RESTART);
-	} else {
-		self->auth_method = "nvm_authenticate";
 	}
 
-	/* success */
 	return TRUE;
 }
 
+static gboolean
+fu_thunderbolt_device_setup_retimer (FuDevice *device, GError **error)
+{
+	FuThunderboltDevice *self = FU_THUNDERBOLT_DEVICE (device);
+	guint16 did;
+	guint16 vid;
+	g_autofree gchar *idx = g_path_get_basename (self->devpath);
+	g_autofree gchar *instance = NULL;
+
+	fu_device_set_physical_id (device, idx);
+	fu_device_set_name (device, "Retimer");
```

Comment:
I'm not sure the user is going to know what that actually is... maybe also `summary` here?

Reply:
I added a summary, quoting the PCIe spec